packages/desktop-client/src/components/manager/WelcomeScreen.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import * as actions from 'loot-core/src/client/actions';
+import {
+  View,
+  Button,
+  Text,
+  P,
+  ExternalLink,
+} from 'loot-design/src/components/common';
+import { colors, styles } from 'loot-design/src/style';
+
+function WelcomeScreen({ createBudget, pushModal }) {
+  return (
+    <View
+      style={{
+        gap: 10,
+        maxWidth: 500,
+        fontSize: 15,
+        maxHeight: '100vh',
+        marginBlock: 20,
+      }}
+    >
+      <Text style={styles.veryLargeText}>Let’s get started!</Text>
+      <View
+        style={{
+          overflowY: 'auto',
+        }}
+      >
+        <P>
+          Actual is a personal finance tool that focuses on beautiful design and
+          a slick user experience.{' '}
+          <strong>Editing your data should be as fast as possible.</strong> On
+          top of that, we want to provide powerful tools to allow you to do
+          whatever you want with your data.
+        </P>
+        <P>
+          Currently Actual implements budgeting based on a{' '}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏maybe this entire paragraph should be dropped. WDYT?

Reply:
Updated and reorganized it a bit, let me know what you think!